packages/garbo/src/familiar/itemFamiliar.ts

```diff
@@ -61,7 +61,7 @@ export function bestFairy(): Familiar {
     }
 
     const bonuses = [
-      ...menu({ includeExperienceFamiliars: false }),
+      ...menu({ includeExperienceFamiliars: true }),
```

Comment:
```suggestion
      ...menu(),
```
including experience familiars is the default, so we can safely do this.